cirq-google/cirq_google/engine/simulated_local_processor.py

```diff
@@ -131,7 +131,7 @@ def get_device(self, gate_sets: Iterable['Serializer'] = ()) -> cirq.Device:
         """Returns a `Device` created from the processor's device specification.
 
         This method queries the processor to retrieve the device specification,
-        which is then use to create a `SerializableDevice` that will validate
+        which is then use to create a `Device` that will validate
```

Comment:
`cirq.Device` so that it links in api docs to Device class

Reply:
Fixed, thank you for catching that!